### ISSUE TITLE
Remove copy to clipboard button

### ISSUE THREAD

**Issue by josh-griffin** [openmsupply/openmsupply-client#870]
Just thought I'd put this here as I had mSupply open and pushed the button!

The mSupply method for a transaction to clipboard: https://github.com/sussol/msupply/blob/master/Project/Sources/Methods/transaction_to_clipbord.4dm

The output:

```
Transaction type :	Supplier Invoice		Name :	
Invoice Num : 	0		Entry date :	22/02/22

line	location	item code	item	quantity	pack size	batch	expiry	Cost Price	extension

						Sub total	0
						Other Expense	0
						Tax	0
						Total	0
```

Currently we are just json stringifying - a formatted csv is probably more appropriate

**Comment by joshxg** [comment]
You'll notice in the method linked that this doesn't get translated. 

That makes me question if this feature really gets much use. However, if we do need to implement, I'm thinking https://www.npmjs.com/package/papaparse

**Comment by mark-prins** [comment]
The option to export to csv/excel is more useful than tab delimited text on the clipboard surely?

I think we should just remove the action from the detail panel

**Comment by joshxg** [comment]
Ah! I had forgotten about the export button.

Do you think that will be done in the browser or through the API?

**Comment by mark-prins** [comment]
I think it should be the API - could probably knock something up in f/e pretty quickly though.
Surely something like sending a filetype to the 'make me a PDF' endpoint?

**Comment by joshxg** [comment]
Yes that sounds like a good idea to me. Wonder if they will also? 😁

**Comment by mark-prins** [comment]
have now been told that it is useful ( maybe v2 though? )
Perhaps combine with #251 ?

**Comment by mark-prins** [comment]
implemented a csv export method in #251 which could be used to implement this

**Comment by mark-prins** [comment]
merged!
